raiden/transfer/state_change.py

```diff
@@ -560,13 +560,17 @@ def __init__(
             self,
             transaction_hash: TransactionHash,
             canonical_identifier: CanonicalIdentifier,
+            our_onchain_locksroot: Locksroot,
+            partner_onchain_locksroot: Locksroot,
             block_number: BlockNumber,
             block_hash: BlockHash,
     ) -> None:
         super().__init__(transaction_hash, block_number, block_hash)
 
         self.token_network_identifier = canonical_identifier.token_network_address
         self.channel_identifier = canonical_identifier.channel_identifier
+        self.our_onchain_locksroot = our_onchain_locksroot
```

Comment:
This change is quite important and non-trivial to figure out, please add some comments here explaining why we **have** to query the locksroot from the Blockchain

Reply:
Added in `blockchain_event_handler.py`